MiniApp/Classes/Display/Displayer.swift

```diff
@@ -1,6 +1,6 @@
 internal class Displayer {
 
-    func getMiniAppView(miniAppId: String) -> MiniAppDisplayProtocol? {
-        return RealMiniAppView.shared.getMiniAppView(miniAppId: miniAppId)
+    func getMiniAppView(miniAppId: String, messageInterface: MiniAppMessageProtocol) -> MiniAppDisplayProtocol? {
```

Comment:
Is the message interface essential for getting the view from the display?

Reply:
Yes, because javascript interface methods will be implemented in `MiniAppView` class, so we have to pass this object.

Reply:
In Android, @minh-rakuten has created a separate method to allow injection via the contract: `injectJSInterface`. I think we will need to align a bit as it is Public API.

Reply:
I don't understand why we need to have a contract here?  This isn't something that will be done on the native side right?

Reply:
I think this one looks correct. Emphasizes the need/dependency on the consumer.